modules/updating-eus-to-eus-upgrade.adoc

```diff
@@ -0,0 +1,160 @@
+* Verify that MachineConfigPools is unpaused.
+
+.Procedure
+
+. Upgrade any OLM Operators to versions that are compatible with both versions you are upgrading to.
+
+. Verify that all MachineConfigPools display a status of `UPDATED` and no MachineConfigPools display a status of `UPDATING`.
+To view the status of all MachineConfigPools, run the following command:
++
+[source,terminal]
+----
+$ oc get mcp
+----
++
+.Example output
++
+Output is trimmed for clarity:
++
+[source,terminal]
+----
+NAME     CONFIG                                         	UPDATED   UPDATING
+master   rendered-master-ecbb9582781c1091e1c9f19d50cf836c       True  	  False
+worker   rendered-worker-00a3f0c68ae94e747193156b491553d5       True  	  False
+----
+
+. To pause the MachineConfigPools you wish to skip reboots on, run the following commands:
++
+[NOTE]
+====
+You cannot pause the master pool.
+====
++
+[source,terminal]
+----
+$ oc patch mcp/worker --type merge --patch '{"spec":{"paused":true}}'
+$ oc get mcp -o
+----
++
+.Example output
++
+[source,terminal]
+----
+# ./oc patch mcp/worker --type merge --patch '{"spec":{"paused":true}}'
+machineconfigpool.machineconfiguration.openshift.io/worker patched
+
+custom-columns=NAME:.metadata.name,PAUSED:.spec.paused
+NAME 	    PAUSED
+master      false
+worker      true
+----
+
+. To change to the `eus-4.10` channel and upgrade to 4.9, run the following commands:
++
+[source,terminal]
+----
+$ oc adm upgrade channel eus-4.10
+$ oc adm upgrade --to-latest
+----
++
+.Example output
++
+[source,terminal]
+----
+Updating to latest version 4.9.11
+----
+
+. To ensure the 4.9 upgrades is completed successfully retrieve the cluster version, run the following command:
++
+[source,terminal]
+----
+$ oc get clusterversion
+----
++
+.Example output
++
+[source,terminal]
+----
+NAME  	  VERSION  AVAILABLE  PROGRESSING   SINCE   STATUS
+version   4.9.11   True       False         6m29s   Cluster version is 4.9.11
+----
+
+. If necessary, upgrade OLM operators using the Administrator perspective on the web console.
+
+. To upgrade to 4.10, run the following command:
++
+[source,terminal]
+----
+$ oc adm upgrade --to-latest
+----
+
+. To ensure the 4.10 upgrade is completed successfully retrieve the cluster version, run the following command:
++
+[source,terminal]
+----
+$ oc get clusterversion
+----
++
+.Example output
++
+[source,terminal]
+----
+NAME  	  VERSION  AVAILABLE  PROGRESSING   SINCE   STATUS
+version   4.10.1   True       False         6m29s   Cluster version is 4.10.1
+----
+
+. To unpause all previously paused MachineConfigPools, run the following command:
++
+[source,terminal]
+----
+$ oc patch mcp/worker --type merge --patch '{"spec":{"paused":false}}'
+----
++
+.Example output
++
+[source,terminal]
+----
+# ./oc patch mcp/worker --type merge --patch '{"spec":{"paused":false}}'
```

Comment:
@sagidlow Looks like we could remove line 135 if we keep line 128. They are duplicated.